### PR TITLE
[android] size vertically based on children

### LINKED ISSUE #8600: [android] Cut text in bookmarks list with a larger system font size
This issue should be easy to fix.

image

### PR DESCRIPTION
**Pull request by nicpottier**
Something simpler. ;)  Fixes #8600 

Original:
  Screenshot 2024-08-15 at 2 19 32 PM

Fixed:
  Screenshot 2024-08-15 at 2 28 48 PM

Fixed (normal fonts):   
  Screenshot 2024-08-15 at 2 29 10 PM

### CODE REVIEW

File: android/app/src/main/res/layout/item_bookmark_category.xml [hunk 1]
```diff
@@ -4,16 +4,17 @@
...
-      android:background="@null"
-      android:buttonTint="?accentColorSelector" />
+    android:id="@+id/checkbox"
+    android:layout_width="@dimen/margin_double_plus"
+    android:layout_height="match_parent"
+    android:layout_centerVertical="true"
+    android:layout_marginStart="@dimen/margin_quarter"
+    android:background="@null"
+    android:button="@drawable/button_visibility_centre_inset"
+    android:buttonTint="?accentColorSelector" />
```
Comment (nicpottier) [inline android/app/src/main/res/layout/item_bookmark_category.xml:17]: Larger diff is due to mixed tab spaces in the original file, I reformatted to 2 space tabs per standards which introduced a few reorderings in the process.

**Comment by nicpottier** [comment]
On second look maybe this is better with these aligned to the top?

Screenshot 2024-08-15 at 2 51 14 PM
Screenshot 2024-08-15 at 2 50 28 PM

**Comment by nicpottier** [comment]
If we'd rather have center I can change it back and change the other options (`CREATE A NEW LIST`) to act accordingly.

**Comment by biodranik** [comment]
Thanks, this is an important improvement, and it definitely looks better.

1. Eyes are important to be detected as clickable. It's better to center them.
2. Other icons also should be centered, otherwise, they look ugly.
3. Is it possible to visually separate bottom buttons from lists?

**Comment by nicpottier** [comment]
Everything centered:
Screenshot 2024-08-15 at 3 21 07 PM

**Comment by nicpottier** [comment]
We could use a different background color for the actions perhaps?
image

**Comment by biodranik** [comment]
Both lists and buttons are pressable, using different appearances may look confusing. Let's finish the initial issue then ([this looks good](https://github.com/organicmaps/organicmaps/pull/8978#issuecomment-2292402404)), and make a separate buttons/list splitting issue for later.

**Comment by nicpottier** [comment]
I never checked in the color changes so PR is at this state right now, just fixing height and centering icons:

Screenshot 2024-08-16 at 6 58 11 AMScreenshot 2024-08-16 at 6 57 46 AM

FWIW, Android does seem to be fine putting buttons in lists with very little differentiation (settings is full of it)

File: android/app/src/main/res/layout/item_bookmark_category.xml (hunk 1 above)
Comment (biodranik) [reply android/app/src/main/res/layout/item_bookmark_category.xml:17]: GitHub can nicely hide whitespaces:
выява

**Review by biodranik — APPROVED** [review]
Thank you for your contribution! Welcome to the club )

**Pull request merged**